risc0/zkvm/Cargo.toml

```diff
@@ -143,3 +142,6 @@ std = [
   "risc0-zkp/std",
   "serde/std",
 ]
+
+# this is a temporary feature that will be removed after proof of fault becomes more stable
+enable-fault-proof = []
```

Comment:
```suggestion
fault-proof = []
```

Reply:
I ended up using both enable-fault-proof and fault-proof... See https://github.com/risc0/risc0/pull/824/commits/cc743477da3eaf08a167a023a553267c6a198871